doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -3995,6 +3995,27 @@ The fully expanded example above (without environment variables) looks like this
 
   curl "https://demo.dataverse.org/api/info/server"
 
+Get Dataverse OpenAPI Specification
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Get the Dataverse OpenAPI specification on the requested format (json or yaml).
+
+.. note:: See :ref:`curl-examples-and-environment-variables` if you are unfamiliar with the use of export below.
+
+.. code-block:: bash
+
+  export SERVER_URL=https://demo.dataverse.org
+  export FORMAT=json
```

Comment:
What's more common, to look at OpenAPI in YAML or JSON? The Payara version was YAML-only, I think. Should we switch this to YAML?